nav-app/src/app/layer-information/layer-information.component.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../colors.scss";
+
+.layer-dialog {
+  font-size: 10pt;
+  padding: 25px;
+
+  .content {
+    overflow-y: scroll;
+    max-height: 75vh;
+
+    p, ul {
+      margin-block-start: 1em;
+      margin-block-end: 1em;
+    }
+
+    code {
+      color: black;
+      padding: 1px 2px;
+      word-break: break-word;
+    }
+
+    table {
+      td {
+        border-top: 1px solid $panel-dark;
+      }
+
+      td:not(td:last-child) {
+        border-right: 1px solid $panel-dark;
+      }
+    }
```

Comment:
Should add border-collapse to greatly improve the table aesthetics:
### Before
![Screen Shot 2021-07-13 at 10 00 33 AM](https://user-images.githubusercontent.com/28868888/125465340-0f40087a-c2eb-48bb-93b3-07e91d3d95c7.png)

### After
![Screen Shot 2021-07-13 at 10 00 39 AM](https://user-images.githubusercontent.com/28868888/125465346-e7e52d2b-736b-4fee-b289-8b54cb9433bd.png)

Reply:
Also increasing padding:
![Screen Shot 2021-07-13 at 10 03 15 AM](https://user-images.githubusercontent.com/28868888/125465894-82e785e3-95e6-4f7f-94d4-06c309abae23.png)

Reply:
And (possibly?) removing the column dividers.

![Screen Shot 2021-07-13 at 10 03 49 AM](https://user-images.githubusercontent.com/28868888/125465936-c6424a2a-98f0-4ac4-992f-e6a2981747fe.png)

Reply:
Note that padding needs to be applied to both `th` and `td`, I forgot to apply it to `th` in my above screenshots.

See [Butterick's Practical Typography](https://practicaltypography.com/tables.html) for some tips and tricks for making aesthetic tables